src/pytest_plugins/filler/ported_tests.py

```diff
@@ -0,0 +1,161 @@
+        "ported_from", "Arguments for showing ported_from marker information"
+    )
+    ported_from_group.addoption(
+        "--show-ported-from",
+        action="store",
+        dest="show_ported_from",
+        default=None,
+        nargs="?",
+        const="paths",
+        help=(
+            "Show information from @pytest.mark.ported_from markers. "
+            "Use '--show-ported-from' or '--show-ported-from=paths' to show static filler paths. "
+            "Use '--show-ported-from=prs' to show PR URLs."
+        ),
+    )
+    ported_from_group.addoption(
+        "--links-as-filled",
+        action="store_true",
+        dest="links_as_filled",
+        default=False,
+        help=(
+            "Convert URLs or paths to filled test file paths for coverage script. "
+            "Used in combination with --show-ported-from."
+        ),
+    )
+
+
+@pytest.hookimpl(tryfirst=True)
+def pytest_configure(config):
+    """Register the plugin if the CLI option is provided."""
+    if config.getoption("show_ported_from"):
+        config.pluginmanager.register(PortedFromDisplay(config), "ported-from-display")
+
+
+class PortedFromDisplay:
+    """Pytest plugin class for displaying ported_from marker information."""
+
+    def __init__(self, config) -> None:
+        """Initialize the plugin with the given pytest config."""
+        self.config = config
+        self.show_mode = config.getoption("show_ported_from")
+
+    @pytest.hookimpl(hookwrapper=True, trylast=True)
+    def pytest_collection_modifyitems(self, session, config, items):
+        """Extract ported_from information from collected test items."""
+        yield
+
+        # Only run on master node when using pytest-xdist
+        if hasattr(config, "workerinput"):
+            return
+
+        paths: Set[str] = set()
+        prs: Set[str] = set()
+
+        for item in items:
+            ported_from_marker = item.get_closest_marker("ported_from")
+            if ported_from_marker:
+                # Extract paths (first positional argument)
+                if ported_from_marker.args:
+                    first_arg = ported_from_marker.args[0]
+                    if isinstance(first_arg, list):
+                        paths.update(first_arg)
+                    elif isinstance(first_arg, str):
+                        paths.add(first_arg)
+
+                # Extract PRs (keyword argument 'pr')
+                if "pr" in ported_from_marker.kwargs:
+                    pr_arg = ported_from_marker.kwargs["pr"]
+                    if isinstance(pr_arg, list):
+                        prs.update(pr_arg)
+                    elif isinstance(pr_arg, str):
+                        prs.add(pr_arg)
+
+        # Output results based on mode
+        if self.show_mode == "prs":
+            output = sorted(prs)
+        else:  # default to "paths"
+            output = sorted(paths)
+
+        if config.getoption("links_as_filled"):
```

Comment:
```suggestion
        if self.links_as_filled:
```
If previous comment is applied.